install/secureblue.butane

```diff
@@ -0,0 +1,46 @@
+variant: fcos
+version: 1.5.0
+passwd:
+  users:
+    - name: core
+      groups:
+        - wheel
+      password_hash: $y$j9T$mefBCJbp/a49aSkTT4hpE1$6BXtrIuV8856t4A9r/R1GW4aR9eKXxsmB8FXt56Hx70 # 'secureblue', change after installation
+      ssh_authorized_keys:
+        - ssh-ed25519 <key>
+storage:
+  files:
+    - path: /opt/install_secureblue.sh
```

Comment:
how does the script end up here?

Reply:
coreos-installer downloads the file. currently unable to identify where in coreos-installer's source where it fetches files directed to be saved.